faq/post_proof.php

```diff
@@ -537,6 +539,12 @@
 once the book has been posted for years.
 </p>
 
+<h3><a name="covers" id="covers"></a>Cover images</h3>
```

Comment:
We should either not have these in here (they shouldn't be needed for the redirect to work) or they should have valid content. These don't provide value to non-pgdp.net consumers. Ditto `faq/ppv.php`.

Reply:
Would it be less confusing for non-pgdp.net consumers to add valid contend from the existing FAQs in the wiki than to remove the content altogether? Or just label any entries in the ToC as dummy entries for redirects?

Reply:
Copying over data from the wiki FAQs would be workable. I would also be fine with not having the anchors in the code FAQ files at all. None of the edits in this file are necessary for the redirects to work correctly.

Reply:
Thinking about it a bit more, probably the latter. Especially for those I put in `faq/ppv.php`, I didn't even try to position them where they are in the actual PPV guidelines -- there are too many differences. I don't think going back to trying to keep the documents in the code up to date is a place we want to go right now.

Reply:
Superfluous sections removed.